### DISCUSSION TITLE
How to group "focus" windows?

### DISCUSSION THREAD

**Discussion by mattfj10** [i3/i3#5067] [Q&A] [answered]
In this hypothetical scenario I have a screen split into 3 parts from light to left: a web browser, a terminal, and two other applications taking up a single space but put into the "tab" view mode. I have the terminal selected, and I want to also group focus the "browser" window and then put those two into a "tab" view, so that way I can have two windows, both of which are tabbed views. What hotkeys could I do to use this? I don't see anything in my config file about this.

**Comment by Airblader** [comment, marked as answer]
You cannot focus multiple windows. The way you do this is by creating the desired layout around one window and then moving the other window into that container.

**Reply by mattfj10** [reply]
Okay well in that case what would be the way to create this "desired layout" around only one window? Right now the only way to achieve what I'm thinking is to move both the individual windows away from the workspace, then put them into tab mode together, have all them selected with Mod1 + a, then move them all back over to the workspace. So how would I move one window into another as you describe?

**Reply by Airblader** [reply]
If the the windows are next to each other in a H split, focus one and do `split v, layout tabbed`. Then focus the other one and move it into the direction of the first one.
